src/com/sun/ts/tests/jpa/core/annotations/mapkeytemporal/EmbeddedEmployee.java

```diff
@@ -1,8 +1,8 @@
-/*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please use 2018, 2020

Reply:
Yes, this was changed by sed so I missed this.

Reply:
Fixed.